plugins/gatsby-theme-iterative-docs/src/utils/shared/sidebar.js

```diff
@@ -23,12 +23,24 @@
 const { titleCase } = require('title-case')
 const sidebar = require('../../../sidebar')
 
+const uppercaseKeywords = ['dvc', 'cml', 'api', 'mlem', 'ml', 'ldb']
 const PATH_ROOT = '/doc'
 const FILE_ROOT = '/docs/'
 const FILE_EXTENSION = '.md'
 
-function dvcTitleCase(slug) {
-  return titleCase(slug.replace(/dvc/g, 'DVC').replace(/-/g, ' '))
+function uppercaseSlugKeywords(slug) {
+  let newSlug = slug
+  uppercaseKeywords.forEach(word => {
+    const regex = new RegExp(String.raw`${word}`, 'g')
+    newSlug = newSlug.replace(regex, word.toUpperCase())
+  })
+  return newSlug
+}
+
+console.log(uppercaseSlugKeywords('ml-mlem-dvc-test-ldb-api-cml'))
```

Comment:
do we need this?

Reply:
Whoops, that was accidentally committed! Deleted now!